packages/metrics/src/features/metrics/utils.ts

```diff
@@ -479,3 +510,25 @@ export function getPopulation(
 
   return totalPopulation ? totalPopulation[populationYearToConsider] : 0
 }
+
+function getDaysPerYear(fromDate: Date, toDate: Date) {
+  const daysPerYear = []
+  const allDates = eachDayOfInterval({ start: fromDate, end: toDate })
+  let currentYear = getYear(fromDate)
+  let daysInYear = 0
+
+  allDates.forEach((date) => {
+    const year = getYear(date)
+    if (year !== currentYear) {
+      daysPerYear.push(daysInYear)
+      currentYear = year
+      daysInYear = 0
+    }
+
+    daysInYear++
+  })
+
+  daysPerYear.push(daysInYear)
+
+  return daysPerYear
+}
```

Comment:
daysPerYear can be calculated by iterating through the years in range, instead of days
```suggestion
function getDaysPerYear(fromDate: Date, toDate: Date) {
  const daysPerYear: number[] = []

  eachYearOfInterval({ start: fromDate, end: toDate }).forEach((date) => {
    const year = date.getFullYear()
    let rangeStart = startOfYear(new Date(year, 0, 1))
    let rangeEnd = endOfYear(new Date(year, 11, 31))
    if (rangeStart < fromDate) rangeStart = fromDate
    if (rangeEnd > toDate) rangeEnd = toDate
    daysPerYear.push(differenceInDays(rangeEnd, rangeStart) + 1)
  })
  return daysPerYear
}
```